### PR TITLE
cloud: add maintenance notification for monitoring features

### PR DESCRIPTION
**Pull request by qqqdan**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by zhangpeijin-milo — APPROVED** [review]
LGTM

File: tidb-cloud/notification-2024-04-09-monitoring-features-maintenance.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+title: 2024-04-09 TiDB Cloud Monitoring Features Maintenance Notification
+summary: Learn about the details of the TiDB Cloud Monitoring features maintenance on April 9, 2024, such as the maintenance window, reason, and impact.
+---
+
+# [2023-08-31] TiDB Cloud Console Maintenance Notification
```
Comment (qiancai) [inline tidb-cloud/notification-2024-04-09-monitoring-features-maintenance.md:6]:
```suggestion
# [2024-04-09] TiDB Cloud Monitoring Features Notification
```

File: tidb-cloud/notification-2024-04-09-monitoring-features-maintenance.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+This notification describes the details that you need to know about the [TiDB Cloud console](https://tidbcloud.com/) maintenance on April 9, 2024.
```
Comment (qiancai) [inline tidb-cloud/notification-2024-04-09-monitoring-features-maintenance.md:8]:
```suggestion
This notification describes the details that you need to know about TiDB Cloud monitoring features maintenance on April 9, 2024.
```

File: tidb-cloud/notification-2024-04-09-monitoring-features-maintenance.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+## Maintenance window
+
+- Start time: 2024-04-09 08:00 (UTC+0)
+- End time: 2024-04-09 12:00 (UTC+0)
+- Duration: 4 hours
+
+## Impact
+
+### Impacted region scope:
```
Comment (qiancai) [inline tidb-cloud/notification-2024-04-09-monitoring-features-maintenance.md:18]:
```suggestion
### Affected regions:

During the maintenance window, the monitoring features in the following regions will be affected:

```

File: tidb-cloud/notification-2024-04-09-monitoring-features-maintenance.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+## Maintenance window
+
+- Start time: 2024-04-09 08:00 (UTC+0)
+- End time: 2024-04-09 12:00 (UTC+0)
+- Duration: 4 hours
+
+## Impact
+
+### Impacted region scope:  
+- Monitoring features for TiDB Dedicated clusters in the following regions：
```
Comment (qiancai) [inline tidb-cloud/notification-2024-04-09-monitoring-features-maintenance.md:19]:
```suggestion
- TiDB Dedicated clusters：
```

File: tidb-cloud/notification-2024-04-09-monitoring-features-maintenance.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+    - Cloud Provider: AWS, Region: Seoul (ap-northeast-2)
+    - Cloud Provider: AWS, Region: Frankfurt (eu-central-1)
+    - Cloud Provider: AWS, Region: Oregon (us-west-2)
+    - Cloud Provider: Google Cloud, Region: Oregon (us-west1)
+    - Cloud Provider: Google Cloud, Region: Tokyo (asia-northeast1)
+    - Cloud Provider: Google Cloud, Region: Singapore (asia-southeast1)
+    - Cloud Provider: Google Cloud, Region: Iowa (us-central1)
+    - Cloud Provider: Google Cloud, Region: Taiwan (asia-east1)
+
+- Monitoring features for Serverless clusters in the following regions：
```
Comment (qiancai) [inline tidb-cloud/notification-2024-04-09-monitoring-features-maintenance.md:30]:
```suggestion
- TiDB Serverless clusters：
```

File: tidb-cloud/notification-2024-04-09-monitoring-features-maintenance.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+    - Cloud Provider: AWS, Region: Frankfurt (eu-central-1)
+    - Cloud Provider: AWS, Region: Oregon (us-west-2)
+
+### Affected Monitoring features ：
```
Comment (qiancai) [inline tidb-cloud/notification-2024-04-09-monitoring-features-maintenance.md:34]:
```suggestion
### Affected monitoring features
```

File: tidb-cloud/notification-2024-04-09-monitoring-features-maintenance.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+
+> **Note:**
+>
+> The maintenance only affects the Monitoring features in the TiDB cluster. All the other functionalities remain unaffected. You can continue to manage the TiDB cluster and perform read/write operations or other operations as usual.
```
Comment (qiancai) [inline tidb-cloud/notification-2024-04-09-monitoring-features-maintenance.md:38]:
```suggestion
> The maintenance only affects the monitoring features in the TiDB cluster. All the other functionalities remain unaffected. You can continue to manage the TiDB cluster and perform read/write operations or other operations as usual.
```

File: tidb-cloud/notification-2024-04-09-monitoring-features-maintenance.md [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+
+- Metrics page will be temporarily unavailable for several short periods（less than 20 mins）.
+- Slow Query page will be temporarily unavailable for several short periods （less than 5 mins）.
+- Metrics Integration with Prometheus, DataDog and NewRelic may have breakpoints
```
Comment (qiancai) [inline tidb-cloud/notification-2024-04-09-monitoring-features-maintenance.md:42]:
```suggestion
- The **Metrics** page will be temporarily unavailable for several short periods (less than 20 mins).
- The **Slow Query** page will be temporarily unavailable for several short periods (less than 5 mins).
- The metrics integration with Prometheus, DataDog, and NewRelic might have breakpoints.
```

File: tidb-cloud/notification-2024-04-09-monitoring-features-maintenance.md [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+
```
Comment (qiancai) [inline tidb-cloud/notification-2024-04-09-monitoring-features-maintenance.md:43]: several short periods (less than 20 mins), 是有几次短暂不可用的时间，每次不超过 20 分钟吗

File: tidb-cloud/notification-2024-04-16-monitoring-features-maintenance.md [hunk 10]
```diff
@@ -0,0 +1,43 @@
+---
+title: 2024-04-16 TiDB Cloud Monitoring Features Maintenance Notification
+summary: Learn about the details of the TiDB Cloud Monitoring features maintenance on April 16, 2024, such as the maintenance window, reason, and impact.
```
Comment (qiancai) [inline tidb-cloud/notification-2024-04-16-monitoring-features-maintenance.md:3]:
```suggestion
summary: Learn about the details of the TiDB Cloud monitoring features maintenance on April 16, 2024, such as the maintenance window, reason, and impact.
```

File: tidb-cloud/notification-2024-04-16-monitoring-features-maintenance.md [hunk 11]
```diff
@@ -0,0 +1,43 @@
...
+---
+
+# [2023-08-31] TiDB Cloud Console Maintenance Notification
```
Comment (qiancai) [inline tidb-cloud/notification-2024-04-16-monitoring-features-maintenance.md:6]:
```suggestion
# [2024-04-16] TiDB Cloud Monitoring Features Notification
```

File: tidb-cloud/notification-2024-04-16-monitoring-features-maintenance.md [hunk 12]
```diff
@@ -0,0 +1,43 @@
...
+
+This notification describes the details that you need to know about the [TiDB Cloud console](https://tidbcloud.com/) maintenance on April 16, 2024.
```
Comment (qiancai) [inline tidb-cloud/notification-2024-04-16-monitoring-features-maintenance.md:8]:
```suggestion
This notification describes the details that you need to know TiDB Cloud monitoring features maintenance on April 16, 2024.
```

File: tidb-cloud/notification-2024-04-16-monitoring-features-maintenance.md [hunk 13]
```diff
@@ -0,0 +1,43 @@
...
+
+## Maintenance window
+
+- Start time: 2024-04-16 08:00 (UTC+0)
+- End time: 2024-04-16 12:00 (UTC+0)
+- Duration: 4 hours
+
+## Impact
+
+### Impacted region scope:
```
Comment (qiancai) [inline tidb-cloud/notification-2024-04-16-monitoring-features-maintenance.md:18]:
```suggestion
### Affected regions:
	
	During the maintenance window, the monitoring features in the following regions will be affected:

```

File: tidb-cloud/notification-2024-04-16-monitoring-features-maintenance.md [hunk 14]
```diff
@@ -0,0 +1,43 @@
...
+## Maintenance window
+
+- Start time: 2024-04-16 08:00 (UTC+0)
+- End time: 2024-04-16 12:00 (UTC+0)
+- Duration: 4 hours
+
+## Impact
+
+### Impacted region scope:  
+- Monitoring features for TiDB Dedicated clusters in the following regions：
```
Comment (qiancai) [inline tidb-cloud/notification-2024-04-16-monitoring-features-maintenance.md:19]:
```suggestion
- TiDB Dedicated clusters：
```

File: tidb-cloud/notification-2024-04-16-monitoring-features-maintenance.md [hunk 15]
```diff
@@ -0,0 +1,43 @@
...
+    - Cloud Provider: AWS, Region: Tokyo (ap-northeast-1)
+    - Cloud Provider: AWS, Region: N. Virginia (us-east-1)
+
+- Monitoring features for Serverless clusters in the following regions：
```
Comment (qiancai) [inline tidb-cloud/notification-2024-04-16-monitoring-features-maintenance.md:23]:
```suggestion
- TiDB Serverless clusters：
```

File: tidb-cloud/notification-2024-04-16-monitoring-features-maintenance.md [hunk 16]
```diff
@@ -0,0 +1,43 @@
...
+    - Cloud Provider: AWS, Region: Tokyo (ap-northeast-1)
+    - Cloud Provider: AWS, Region: N. Virginia (us-east-1)
+
+### Affected Monitoring features ：
```
Comment (qiancai) [inline tidb-cloud/notification-2024-04-16-monitoring-features-maintenance.md:27]:
```suggestion
### Affected monitoring features
```

File: tidb-cloud/notification-2024-04-16-monitoring-features-maintenance.md [hunk 17]
```diff
@@ -0,0 +1,43 @@
...
+
+> **Note:**
+>
+> The maintenance only affects the Monitoring features in the TiDB cluster. All the other functionalities remain unaffected. You can continue to manage the TiDB cluster and perform read/write operations or other operations as usual.
```
Comment (qiancai) [inline tidb-cloud/notification-2024-04-16-monitoring-features-maintenance.md:31]:
```suggestion
> The maintenance only affects the monitoring features in the TiDB cluster. All the other functionalities remain unaffected. You can continue to manage the TiDB cluster and perform read/write operations or other operations as usual.
```

File: tidb-cloud/notification-2024-04-16-monitoring-features-maintenance.md [hunk 18]
```diff
@@ -0,0 +1,43 @@
...
+
+- Metrics page will be temporarily unavailable for several short periods（less than 20 mins）.
+- Slow Query page will be temporarily unavailable for several short periods （less than 5 mins）.
+- Metrics Integration with Prometheus, DataDog and NewRelic may have breakpoints
```
Comment (qiancai) [inline tidb-cloud/notification-2024-04-16-monitoring-features-maintenance.md:35]:
```suggestion
- The **Metrics** page will be temporarily unavailable for several short periods (less than 20 mins).
- The **Slow Query** page will be temporarily unavailable for several short periods (less than 5 mins).
- The metrics integration with Prometheus, DataDog, and NewRelic might have breakpoints.
```

File: tidb-cloud/notification-2024-04-09-monitoring-features-maintenance.md (hunk 9 above)
Comment (qqqdan) [reply tidb-cloud/notification-2024-04-09-monitoring-features-maintenance.md:43]: 是的

File: tidb-cloud/notification-2024-04-09-monitoring-features-maintenance.md [hunk 19]
```diff
@@ -0,0 +1,53 @@
...
+    - Cloud Provider: AWS, Region: Oregon (us-west-2)
+
+### Affected monitoring features
+
+> **Note:**
+>
+> The maintenance only affects the monitoring features in the TiDB cluster. All the other functionalities remain unaffected. You can continue to manage the TiDB cluster and perform read/write operations or other operations as usual.
+
+- The **Metrics** page will be temporarily unavailable for several short periods (less than 20 mins).
+- The **Slow Query** page will be temporarily unavailable for several short periods (less than 5 mins).
```
Comment (qiancai) [inline tidb-cloud/notification-2024-04-09-monitoring-features-maintenance.md:44]:
```suggestion
- The **Metrics** page will be temporarily unavailable for several short periods (each less than 20 mins).
- The **Slow Query** page will be temporarily unavailable for several short periods (each less than 5 mins).
```

File: tidb-cloud/notification-2024-04-16-monitoring-features-maintenance.md [hunk 20]
```diff
@@ -0,0 +1,46 @@
...
+    - Cloud Provider: AWS, Region: N. Virginia (us-east-1)
+
+### Affected monitoring features
+
+> **Note:**
+>
+> The maintenance only affects the monitoring features in the TiDB cluster. All the other functionalities remain unaffected. You can continue to manage the TiDB cluster and perform read/write operations or other operations as usual.
+
+- The **Metrics** page will be temporarily unavailable for several short periods (less than 20 mins).
+- The **Slow Query** page will be temporarily unavailable for several short periods (less than 5 mins).
```
Comment (qiancai) [inline tidb-cloud/notification-2024-04-16-monitoring-features-maintenance.md:37]:
```suggestion
- The **Metrics** page will be temporarily unavailable for several short periods (each less than 20 mins).
- The **Slow Query** page will be temporarily unavailable for several short periods (each less than 5 mins).
```

File: tidb-cloud/notification-2024-04-16-monitoring-features-maintenance.md [hunk 21]
```diff
@@ -0,0 +1,46 @@
+---
+title: 2024-04-16 TiDB Cloud Monitoring Features Maintenance Notification
+summary: Learn about the details of the TiDB Cloud monitoring features maintenance on April 16, 2024, such as the maintenance window, reason, and impact.
+---
+
+# [2024-04-16] TiDB Cloud Monitoring Features Notification
+
+This notification describes the details that you need to know TiDB Cloud [monitoring features](/tidb-cloud/monitor-tidb-cluster.md)maintenance on April 16, 2024.
```
Comment (qiancai) [inline tidb-cloud/notification-2024-04-16-monitoring-features-maintenance.md:8]:
```suggestion
This notification describes the details that you need to know TiDB Cloud [monitoring features](/tidb-cloud/monitor-tidb-cluster.md) maintenance on April 16, 2024.
```

File: tidb-cloud/notification-2024-04-09-monitoring-features-maintenance.md [hunk 22]
```diff
@@ -0,0 +1,53 @@
...
+
+## Maintenance window
+
+- Start time: 2024-04-09 08:00 (UTC+0)
+- End time: 2024-04-09 12:00 (UTC+0)
+- Duration: 4 hours
+
+## Impact
+
+### Affected regions:
```
Comment (qiancai) [inline tidb-cloud/notification-2024-04-09-monitoring-features-maintenance.md:18]:
```suggestion
### Affected regions
```

File: tidb-cloud/notification-2024-04-16-monitoring-features-maintenance.md [hunk 23]
```diff
@@ -0,0 +1,46 @@
...
+
+## Maintenance window
+
+- Start time: 2024-04-16 08:00 (UTC+0)
+- End time: 2024-04-16 12:00 (UTC+0)
+- Duration: 4 hours
+
+## Impact
+
+### Affected regions:
```
Comment (qiancai) [inline tidb-cloud/notification-2024-04-16-monitoring-features-maintenance.md:18]:
```suggestion
### Affected regions
```

File: tidb-cloud/notification-2024-04-16-monitoring-features-maintenance.md [hunk 24]
```diff
@@ -0,0 +1,46 @@
...
+
+- Start time: 2024-04-16 08:00 (UTC+0)
+- End time: 2024-04-16 12:00 (UTC+0)
+- Duration: 4 hours
+
+## Impact
+
+### Affected regions
+	
+	During the maintenance window, the monitoring features in the following regions will be affected:
```
Comment (qiancai) [inline tidb-cloud/notification-2024-04-16-monitoring-features-maintenance.md:20]:
```suggestion
During the maintenance window, the monitoring features in the following regions will be affected:
```

**Review by hfxsd — APPROVED** [review]

File: tidb-cloud/notification-2024-04-09-monitoring-features-maintenance.md [hunk 25]
```diff
@@ -0,0 +1,53 @@
+---
+title: 2024-04-09 TiDB Cloud Monitoring Features Maintenance Notification
+summary: Learn about the details of TiDB Cloud monitoring features maintenance on April 9, 2024, such as the maintenance window, reason, and impact.
+---
+
+# [2024-04-09] TiDB Cloud Monitoring Features Notification
```
Comment (hfxsd) [inline tidb-cloud/notification-2024-04-09-monitoring-features-maintenance.md:6]:
```suggestion
# [2024-04-09] TiDB Cloud Monitoring Features Maintenance Notification
```

File: tidb-cloud/notification-2024-04-16-monitoring-features-maintenance.md [hunk 26]
```diff
@@ -0,0 +1,46 @@
+---
+title: 2024-04-16 TiDB Cloud Monitoring Features Maintenance Notification
+summary: Learn about the details of the TiDB Cloud monitoring features maintenance on April 16, 2024, such as the maintenance window, reason, and impact.
+---
+
+# [2024-04-16] TiDB Cloud Monitoring Features Notification
```
Comment (hfxsd) [inline tidb-cloud/notification-2024-04-16-monitoring-features-maintenance.md:6]:
```suggestion
# [2024-04-16] TiDB Cloud Monitoring Features Maintenance Notification
```

File: tidb-cloud/notification-2024-04-09-monitoring-features-maintenance.md (hunk 25 above)
Comment (qiancai) [inline tidb-cloud/notification-2024-04-09-monitoring-features-maintenance.md:6]:
```suggestion
# [2024-04-09] TiDB Cloud Monitoring Features Maintenance Notification
```

File: tidb-cloud/notification-2024-04-16-monitoring-features-maintenance.md (hunk 26 above)
Comment (qiancai) [inline tidb-cloud/notification-2024-04-16-monitoring-features-maintenance.md:6]:
```suggestion
# [2024-04-16] TiDB Cloud Monitoring Features Maintenance Notification
```

**Review by qiancai — APPROVED** [review]

File: tidb-cloud/notification-2024-04-09-monitoring-features-maintenance.md [hunk 27]
```diff
@@ -0,0 +1,53 @@
+---
+title: 2024-04-09 TiDB Cloud Monitoring Features Maintenance Notification
+summary: Learn about the details of TiDB Cloud monitoring features maintenance on April 9, 2024, such as the maintenance window, reason, and impact.
```
Comment (qiancai) [inline tidb-cloud/notification-2024-04-09-monitoring-features-maintenance.md:3]:
```suggestion
summary: Learn about the details of the TiDB Cloud monitoring features maintenance on April 9, 2024, such as the maintenance window, reason, and impact.
```

File: tidb-cloud/notification-2024-04-09-monitoring-features-maintenance.md [hunk 28]
```diff
@@ -0,0 +1,53 @@
...
+    - Cloud Provider: AWS, Region: Oregon (us-west-2)
+
+### Affected monitoring features
+
+> **Note:**
+>
+> The maintenance only affects monitoring features in the TiDB cluster. All the other functionalities remain unaffected. You can continue to manage the TiDB cluster and perform read/write operations or other operations as usual.
+
+- The **Metrics** page will be temporarily unavailable for several short periods (each less than 20 mins).
+- The **Slow Query** page will be temporarily unavailable for several short periods (each less than 5 mins).
```
Comment (qiancai) [inline tidb-cloud/notification-2024-04-09-monitoring-features-maintenance.md:44]:
```suggestion
- The **Metrics** page will be temporarily unavailable for several short periods (each less than 20 minutes).
- The **Slow Query** page will be temporarily unavailable for several short periods (each less than 5 minutes).
```

File: tidb-cloud/notification-2024-04-16-monitoring-features-maintenance.md [hunk 29]
```diff
@@ -0,0 +1,46 @@
...
+    - Cloud Provider: AWS, Region: N. Virginia (us-east-1)
+
+### Affected monitoring features
+
+> **Note:**
+>
+> The maintenance only affects monitoring features in the TiDB cluster. All the other functionalities remain unaffected. You can continue to manage the TiDB cluster and perform read/write operations or other operations as usual.
+
+- The **Metrics** page will be temporarily unavailable for several short periods (each less than 20 mins).
+- The **Slow Query** page will be temporarily unavailable for several short periods (each less than 5 mins).
```
Comment (qiancai) [inline tidb-cloud/notification-2024-04-16-monitoring-features-maintenance.md:37]:
```suggestion
- The **Metrics** page will be temporarily unavailable for several short periods (each less than 20 minutes).
- The **Slow Query** page will be temporarily unavailable for several short periods (each less than 5 minutes).
```

**Pull request merged**